java/client/src/main/java/com/youtube/vitess/client/cursor/Cursor.java

```diff
@@ -40,191 +31,32 @@
  * accessed concurrently without additional synchronization.
  */
 public abstract class Cursor implements AutoCloseable {
-  public abstract long getRowsAffected() throws SQLException;
+  private Map<String, Integer> fieldMap;
 
-  public abstract long getInsertId() throws SQLException;
+  abstract public long getRowsAffected() throws SQLException;
```

Comment:
Yes definitely. I assumed he reversed them because he knows the style guide better than I do, but I guess I should have checked. :)

@erzel feel free to send me a quick PR if you have a sec. Otherwise I'll make a note to get to it later.